### PR TITLE
Added some comments to the models

### PR DESCRIPTION
**Pull request by mphprogrammer**
Added comments to the models, the Controllers and views are straight forward. There are also comments in the config files and initializers. However, these have been incorporated from the git-go.

### CODE REVIEW

File: app/models/data_csv.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 require 'csv'
 
+###############################################################################
+## DataCsv
+## Holds the merrged data from CSV-sourced data tables, as well as methods to
```
Comment (ayaleloehr) [inline app/models/data_csv.rb]: `sed 's/merrged/merged'`

File: app/models/data_csv.rb [hunk 2]
```diff
@@ -299,6 +305,7 @@ def self.initialize_with_weams
 
     names = Weam::USE_COLUMNS.map(&:to_s).join(', ')
 
+    # Include only those schools marked as approved (c.f., Weam model)
```
Comment (ayaleloehr) [inline app/models/data_csv.rb]: what's c.f.?

File: app/models/data_csv.rb [hunk 3]
```diff
@@ -575,11 +605,12 @@ def self.update_with_scorecard
 
   ###########################################################################
   ## update_with_ipeds_ic
-  ## Updates the DataCsv table with data from the scorecards table.
+  ## Updates the DataCsv table with data from the IC table.
```
Comment (ayaleloehr) [inline app/models/data_csv.rb]: Is IC a known acronym or defined elsewhere?

File: app/models/data_csv.rb [hunk 4]
```diff
@@ -590,11 +621,12 @@ def self.update_with_ipeds_ic
 
   ###########################################################################
   ## update_with_ipeds_hd
-  ## Updates the DataCsv table with data from the scorecards table.
+  ## Updates the DataCsv table with data from the HD table.
```
Comment (ayaleloehr) [inline app/models/data_csv.rb]: Same as above about `IC`. Is `HD` defined elsewhere.

File: app/models/data_csv.rb [hunk 5]
```diff
@@ -804,11 +837,17 @@ def self.update_with_settlement
...
   def self.update_with_hcm
+    # Sets caution flags if the corresponing approved school (by OPE6 id) 
+    # has an entry in the Hcm table.
     query_str = 'UPDATE data_csvs SET '
     query_str += 'caution_flag = TRUE'
     query_str += ' FROM hcms '
     query_str += 'WHERE data_csvs.ope6 = hcms.ope6; '
 
+    # Sets the caution flag reason for each school, by OPE6. The inner select
+    # encloses unique reasons within a lable while the outer select aggregates
```
Comment (ayaleloehr) [inline app/models/data_csv.rb]: lable -> label.

**Comment by ayaleloehr** [comment]
@mphprogrammer only a few small nits, LGTM otherwise. Feel free to merge once you address those small issues. Nice work here, the more we can get out of your head about what you did (and why), the better it will be for this project's long term success, so thanks for taking the time to write it all out!

File: app/models/data_csv.rb (hunk 2 above)
Comment (mphprogrammer) [inline app/models/data_csv.rb]: Its the latin abbreviation for "conferre" meaning refer elsewhere (in this case, the Weam model) for more information.

**Comment by ayaleloehr** [comment]
LGTM.

**Pull request merged**